### PR TITLE
CORE-1721 Fix new tool submissions in EditTool form

### PR DESCRIPTION
**Pull request by psarando**
This PR will fix a new tool submission bug introduced in #454, and also includes some formatting updates from prettier.

### CODE REVIEW

File: src/components/tools/edit/EditTool.js [hunk 1]
```diff
@@ -133,8 +133,8 @@ function EditToolDialog(props) {
     });
 
     const { mutate: addNewTool, status: newToolStatus } = useMutation(
-        ({ submission }) =>
-            isAdmin ? adminAddTool(submission) : addTool(submission),
+        () =>
+            isAdmin ? adminAddTool(toolSubmission) : addTool(toolSubmission),
```
Comment (psarando) [inline src/components/tools/edit/EditTool.js:137]: The submission values were moved into state in this file's last PR.

**Comment by ianmcorvidae** [comment]
As far as I can tell this looks good -- I guess the bit you commented on is the fix for the bug?

**Comment by psarando** [comment]
> I guess the bit you commented on is the fix for the bug?

Yep. I also kept it in its own commit, separate from the formatting commit, if you want to look at it that way too 👍

**Review by slr71 — APPROVED** [review]
Looks good to me! :shipit:

**Comment by psarando** [comment]
Thanks for the reviews! 👍

**Pull request merged**